StreamChat.xcodeproj/project.pbxproj

```diff
@@ -5198,17 +5242,27 @@
 		A57E5C7624C6FA3900187DC2 /* ReleaseTests */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ARCHS = (
+					arm64,
+					armv7,
+					armv7s,
+					i386,
+					x86_64,
+				);
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = D48F367GRM;
+				DEVELOPMENT_TEAM = EHV7XZLAHA;
```

Comment:
Did you mean to commit this?

Reply:
Without this change I was not able to build macOS, issue with macOS dev sertificates

Reply:
Seems like it was changed to correct team though?